pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -341,6 +341,33 @@ self: super:
           "/rdrand/rngAddEntropy/"
         ]);
     }) super.botan-low;
+
+    # 2025-11-07: Broken since 2015 - https://github.com/NixOS/nixpkgs/pull/459465
+    hamid = markBroken super.hamid;
+    # 2025-11-07: Broken since 2016 - https://github.com/NixOS/nixpkgs/pull/459465
+    huckleberry = markBroken super.huckleberry;
+    log = markBroken super.log;
+    # 2025-11-07: Broken since 2017 - https://github.com/NixOS/nixpkgs/pull/459465
+    gtk-traymanager = markBroken super.gtk-traymanager;
+    pipes-zlib = markBroken super.pipes-zlib;
+    pthread = markBroken super.pthread;
+    # 2025-11-07: Broken since 2018 - https://github.com/NixOS/nixpkgs/pull/459465
+    system-linux-proc = markBroken super.system-linux-proc;
+    # 2025-11-07: Broken since 2021 - https://github.com/NixOS/nixpkgs/pull/459465
+    gerrit = markBroken super.gerrit;
+    # 2025-11-07: Broken since 2023 - https://github.com/NixOS/nixpkgs/pull/459465
+    copilot-c99 = markBroken super.copilot-c99;
```

Comment:
(This is also pretty simple to work around, it just assumes `gcc` is available, so the test suite should be gated on `stdenv.cc.isGnu` (and presumably `stdenv.cc.targetPrefix = ""`).)